polaris-react/src/components/Filters/Filters.tsx

```diff
@@ -193,15 +190,15 @@ class FiltersInner extends Component<CombinedProps, State> {
           >
             <div className={styles.FilterTriggerLabelContainer}>
               <h3 className={styles.FilterTriggerTitle}>
-                <TextStyle
-                  variation={
-                    this.props.disabled || filter.disabled
-                      ? 'subdued'
-                      : undefined
-                  }
+                <Text
+                  variant="bodyMd"
+                  as="span"
+                  {...((this.props.disabled || filter.disabled) && {
+                    color: 'subdued',
+                  })}
```

Comment:
Not a blocker, I'd consider a ternary inside of the color prop easier to read:

```suggestion
                  color={this.props.disabled || filter.disabled ? 'subdued' : undefined}
```